cmd/ocm-backplane/cloud/ssm_test.go

```diff
@@ -219,10 +224,11 @@ var _ = Describe("SSM command", func() {
 				err := StartSSMsession(&cobra.Command{}, []string{})
 				Expect(err).ToNot(HaveOccurred())
 
-				// Verify AWS credentials
+				// Verify ENV variables
 				Expect(os.Getenv("AWS_ACCESS_KEY_ID")).To(Equal("TEST_ACCESS_KEY"))
 				Expect(os.Getenv("AWS_SECRET_ACCESS_KEY")).To(Equal("TEST_SECRET_KEY"))
 				Expect(os.Getenv("AWS_SESSION_TOKEN")).To(Equal("TEST_SESSION_TOKEN"))
+				Expect(os.Getenv("HTTPS_PROXY")).To(Equal(fakeProxyURL))
```

Comment:
The `StartSession` API call loads the proxy URL from the backplane configuration. Why all the tests not aligned and instead reads the proxy from env variables?